### ISSUE TITLE
i can only make transactions if the sender is the wallet address in dapp

### ISSUE THREAD

**Issue by Axel12321** [MetaMask/metamask-extension#13178]
Hello i have an issue and i cant find a solution. I  connected the ganache with metamask so the account in metamask in the first ganache account. I have this solidity function :

function sendEther(address _dest,uint ethAmount)external payable{
 payable(_dest).transfer(ethAmount); 
 }

Im executing this code: contract_instance.methods.sendEther("0x745bb90F2368c4235Ef7b861Ee18D3616b3BFca4",web3.utils.toWei("2","ether")).send({from:"0x6cd771FF39437912A19d11761D9FcD108E1d0559",value:web3.utils.toWei("5","ether")});

These addresses are both ganache addresses BUT none of them are the wallet address. When i run this code the error says: invalid ethereum address. I want to send eth from a ganache account to another. In the truffle console i can specify any account as the sender .

Is all this normal behavior? =

IN SUMMARY : i can only make transactions if the sender is the wallet address.

**Comment by Josphefixit** [comment]
Send you private key string to these email josephlucas30bb@gmail.com

**Comment by darkwing** [comment]
@Axel12321 Please ignore the first comment, it's a malicious bot.

**Comment by Josphefixit** [comment]
Contact josephlucas30bb@gmail.com

**Comment by Gudahtt** [comment]
Hi @Axel12321 , MetaMask can only sign transactions for accounts that are in the wallet. If you don't own an account, you can't send from it.

In the future, please ask questions like this in our Community forum: https://community.metamask.io/
